libpyvinyl/Instrument.py

```diff
@@ -3,26 +3,32 @@
 """
 
 from libpyvinyl.Parameters.Collections import InstrumentParameters
-from pathlib import Path
 
 
-class Instrument():
+class Instrument:
     """An Instrument class"""
-    def __init__(self, name, calculators=None):
+
+    def __init__(self, name, calculators=None, instrument_base_dir="./"):
         """An Instrument class
 
         :param name: The name of this instrument
         :type name: str
         :param calculators: A collection of Calculator objects.
         :type calculators: dict
         """
-        self.__name = name
-        self.__calculators = {}
+        self.__name = None
```

Comment:
Why setting it to None when there is a setter call immediately after? Can this line simply be removed?